x-pack/plugins/security_solution/public/flyout/index.tsx

```diff
@@ -98,9 +98,15 @@ const expandableFlyoutDocumentsPanels: ExpandableFlyoutProps['registeredPanels']
  * Flyout used for the Security Solution application
  * We keep the default EUI 1000 z-index to ensure it is always rendered behind Timeline (which has a z-index of 1001)
  */
-export const SecuritySolutionFlyout = memo(() => (
-  <ExpandableFlyout registeredPanels={expandableFlyoutDocumentsPanels} paddingSize="none" />
-));
+export const SecuritySolutionFlyout = memo(() => {
+  return (
+    <ExpandableFlyout
+      registeredPanels={expandableFlyoutDocumentsPanels}
+      paddingSize="none"
+      onClose={() => {}}
```

Comment:
Can we make this prop optional? 
The `onClose` is required by the `EuiFlyout` because it is not controlled, the logic to close it needs to be implemented from outside. For the `ExpandableFlyout` however, this logic is already controlled, so this prop is not strictly necessary, it's only informative.
Making it required will force devs to pass dummy functions like this one when we don't need to know when the flyout is closed.

Reply:
I thought about it but wanted to stay as close as EUI as possible. I can make the change!

Reply:
@semd  is [this](https://github.com/elastic/kibana/pull/183553/commits/61d4ffa4265b580dd6093beaa4c240da82812579) what you had in mind?